bundles/io/org.eclipse.smarthome.io.console/src/main/java/org/eclipse/smarthome/io/console/internal/extension/ItemConsoleCommandExtension.java

```diff
@@ -151,7 +151,11 @@ private void listItems(Console console, String pattern) {
                 console.println(item.toString());
             }
         } else {
-            console.println("No item found for this pattern.");
+            if (pattern == null || (pattern != null && pattern.isEmpty())) {
```

Comment:
Shouldn't this be:

```
if (pattern == null || pattern.isEmpty()) {
```

Reply:
Indeed, thanks. Was too early in the morning. For the fix please see #5645